contrib/devtools/lint-python.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check for specified flake8 warnings in python files.
+
+# F401: module imported but unused
+flake8 --ignore=B,C,E,F,I,N,W --select=F401 .
```

Comment:
Where did you get this list of error classes to ignore? I believe flake8 only uses:
- E and W (from pyflakes)
- F from pycodestyle
- C from mccabe
(see http://flake8.pycqa.org/en/latest/user/error-codes.html)

Where did the B, I and N classes come from?

Reply:
* `B` is used for blind except warnings - see [`flake8-blind-except`](https://pypi.python.org/pypi/flake8-blind-except)
* `I` is used for import warnings - see [`flake8-import-order`](https://github.com/PyCQA/flake8-import-order)
* `N` is used for naming convention warnings - see [`pep8-naming`](https://github.com/PyCQA/pep8-naming)

Reply:
Thanks @practicalswift. I'll add those to my tool belt!